dist/index.js

```diff
@@ -64939,20 +64939,12 @@ const run = async () => {
     const token = core.getInput('token', { required: true })
     const octokit = github.getOctokit(token)
 
-    // Get the repo object, which contains the `default_branch` property.
-    const repoResult = await octokit.request('GET /repos/{owner}/{repo}', { owner, repo })
-    const defaultBranch = repoResult.data.default_branch
-    core.info(`Using default branch '${defaultBranch}'`)
-
-    // Fetch the current SHA of the default branch
-    const ref = `heads/${defaultBranch}`
-    const refResult = await octokit.rest.git.getRef({ owner, repo, ref })
-    core.info(`Found '${defaultBranch}' with SHA ${refResult.data.object.sha}`)
-
```

Comment:
With this removal, I don't think we need the `octokit` variable, which means I don't think we need a token input.

Reply:
Removed and added basic eslint to catch unused vars. 👍